src/main/java/com/dev/cinema/AppMain.java

```diff
@@ -43,9 +51,18 @@ public static void main(String[] args) {
         cinemaHallService.getAll().forEach(System.out::println);
         movieSessionService.findAvailableSessions(1L, today.toLocalDate())
                 .forEach(System.out::println);
+        User Bob = new User();
```

Comment:
what is wrong here?

Reply:
Is Bob missing Alice?^^"

Reply:
'Bob' must match pattern '^[a-z]([a-z0-9][a-zA-Z0-9]*)?$', agree